README.md

```diff
@@ -1,146 +1,25 @@
 # Overview
 
+![tcl-42smartshoppinglist](https://user-images.githubusercontent.com/86992220/170784008-9494285a-d273-4203-8f3d-ad233c6ae778.png)
+
 ## What is this project?
 
-The goal of this project is to collaborate with your Collab Lab team to create a “smart” shopping list app that learns your buying habits and helps you remember what you’re likely to need to buy on your next trip to the store.
+This is a smart shopping app developed by a group of three developers under [The Collab Lab](https://the-collab-lab.codes/about-us/) mentorship program. This sleek app allows users to keep track of the items they purchase and learns the purchase frequency those items. This app is mobile and accessibility (a11y) friendly. 
```

Comment:
Add 'of' to the following sentence: This sleek app allows users to keep track of the items they purchase and learns the purchase frequency **of** those items.